pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -839,21 +846,6 @@ async def check_blocker_bugs(self):
         if int(match[1]) != 0:
             raise VerificationError(f"{int(match[1])} blocker Bug(s) found for release; do not proceed without resolving. See https://art-docs.engineering.redhat.com/release/4.y.z-stream/#handling-blocker-bugs. To permit this validation error, see https://art-docs.engineering.redhat.com/jenkins/build-promote-assembly-readme/#permit-certain-validation-failures. Elliott output: {stdout}")
 
-    async def get_advisory_info(self, advisory: int) -> Dict:
```

Comment:
We have this pattern of calling elliott and doozer commands from pyartcd, even for very simple info like advisory_info, since they were separate repos before. Now that pyartcd can directly make use of elliottlib and doozerlib, these don't make much sense and would need a refactor. I would suggest leaving these as is for now, and come back to them in another PR